hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -297,7 +300,9 @@
                    // Coverage database output location
                    "-cm_dir {cov_db_dir}",
                    // The following option is to improve runtime performance
-                   "-Xkeyopt=rtopt"
+                   "-Xkeyopt=rtopt",
+                   // Exclude FSM transitions that can only happen on reset
+                   "-cm_fsmresetfilter {vcs_fsm_reset_cov_cfg_file}",
```

Comment:
If there are multiple FSMs in the design on different resets, then some IPs may need to supply additional files. I wonder if we should make this a list instead. 

```
  vcs_fsm_reset_cov_cfg_files: [<list of files>]
```

Reply:
Maybe worth checking if VCS allows `-cm_fsmresetfilter file1+file2_file3` just like the `-cm_hier` option.

Reply:
user guide didn't mention this usage. I tried below, it doesn't work.

Since it's just a list of reset signal name, I think we can add more in this common cfg file. Or even ask design to use those reset name that we add here. We may not need to support multiple cfg files.

> Warning-[VCM-FOPENWR] File open error.
  Could not open the file named 
'/workspace/mnt/repo_top/hw/dv/tools/vcs/fsm_reset_cov2.cfg+/workspace/mnt/repo_top/hw/dv/tools/vcs/fsm_reset_cov.cfg'.
  Details of the error : No such file or directory.
  This file is provided by -cm_fsmresetfilter option. Please check if the file
  exists or has read permissions.

Reply:
Hmm, we may not have such cases, and we probably have at most one state machine per module, so changing the reset name seems viable.

Reply:
Okay. We shouldn't be assuming that we will not run into this case though.